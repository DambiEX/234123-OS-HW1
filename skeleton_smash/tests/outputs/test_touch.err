touch: cannot touch '56:48:20:12:8:1/bin/bash': No such file or directory
touch: cannot touch '00:30:14:6:10:2/bin/bash': No such file or directory
