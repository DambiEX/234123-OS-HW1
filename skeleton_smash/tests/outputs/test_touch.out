smash> smash> 2024-03-06 07:19:12.907866147 +0000, 2024-03-06 07:19:12.907866147 +0000
smash> smash> 2024-03-06 07:19:12.919866342 +0000, 2024-03-06 07:19:12.919866342 +0000
smash> 